cluster-api/src/main/java/io/aiven/klaw/clusterapi/utils/ClusterApiUtils.java

```diff
@@ -42,6 +43,7 @@ public class ClusterApiUtils {
   public static final String KAFKA_SR_CREDENTIALS_PROPERTY_SFX = ".klaw.schemaregistry.credentials";
   public static final String KAFKA_CONFLUENT_CLOUD_CREDENTIALS_PROPERTY_SFX =
       ".klaw.confluentcloud.credentials";
+  private static final long TIME_OUT_SECS_FOR_TOPICS = 5;
```

Comment:
Should we make this configurable where larger clusters might need a larger timeout?